.gitmodules

```diff
@@ -22,3 +22,15 @@
 [submodule "sim/toolchain-utilities/riscv-tests"]
 	path = sim/toolchain-utilities/riscv-tests
 	url = https://github.com/riscv-software-src/riscv-tests
+[submodule "sim/rocket-chip"]
+	path = sim/rocket-chip
+	url = https://github.com/chipsalliance/rocket-chip.git
+[submodule "sim/berkeley-hardfloat"]
```

Comment:
Stupid question: did FireSim have a hardfloat dependency?

Reply:
This just copies how Chipyard set up it's RC SBT project. I'm not sure why Chipyard had it's own hardfloat but I figured I would keep it the same.